core-job/src/main/java/org/apache/kylin/job/util/FlatTableSqlQuoteUtils.java

```diff
@@ -54,25 +55,29 @@ private static synchronized void setQuote() {
     }
 
     public static String getQuote() {
-        setQuote();
-        return quote;
+        if (kylinConfig.getJdbcSourceDialect().equals(SourceDialect.POSTGRESQL.source)) {
+            return "";
```

Comment:
It is looks strange here, as far as I know, postgresql us double quote(") to quote identifier.